charmcraft/commands/store/__init__.py

```diff
@@ -1132,8 +1172,13 @@ class UploadResourceCommand(BaseCommand):
         specified charm. This charm needs to have the resource declared
         in its metadata (in a preoviously uploaded to Charmhub revision).
 
-        to the packaging standard. This command will finish successfully
-        once the package is approved by Charmhub.
+        The resource can be a file from your computer (use the '--filepath'
+        option) or an OCI Image (use the '--image' option).
+
+        The OCI image description uses the [organization/]name[:reference]
```

Comment:
The `[:reference]` format description here applies only to `[:tag]`, shouldn't we also mention `[@digest]` here to be consistent with the `--image` parameter help message below?

Reply:
yes! thanks, good catch! improved